test/member_page_test.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/member_page.rb'
+
+describe MemberPage do
+  around { |test| VCR.use_cassette(url.split('/').last, &test) }
+
+  subject do
+    MemberPage.new(response: Scraped::Request.new(url: url).response)
+  end
+
+  describe 'Page with image' do
+    let(:url) { 'http://www.alderney.gov.gg/article/4081/Ian-Tugby' }
+
+    it 'should return the expected data' do
+      subject.to_h.must_equal(
+        id:     '4081',
+        name:   'Ian Tugby',
+        image:  './media/image/6/o/140913_Alderney-801.jpg',
```

Comment:
As a temporary fix, I've gone back to making the URL absolute in the MemberClass. The pre-factor data contains images with absolute URLs.